Sources/PostgresNIO/New/PSQLConnection.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```